containers/ddev-webserver/ddev-webserver-base-files/etc/bashrc/commandline-addons.bashrc

```diff
@@ -1,4 +1,14 @@
-export PATH="~/bin:$PATH:/var/www/html/bin"
+case ":$PATH:" in
+    *":$HOME/bin:"*) ;;
+    *) PATH="$HOME/bin:$PATH" ;;
```

Comment:
So the fix is to write `$HOME/bin` instead of `~/bin:$PATH` just like Debian does.

Reply:
This also needs a comment explaining what it's doing.

But it's great, and an excellent solution.